ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.19.yaml

```diff
@@ -37,6 +37,11 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: installer-rehearse-gcp
+  cron: '@yearly'
+  steps:
+    cluster_profile: gcp-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
```

Comment:
This workflow belongs windows container team, we do not need to test it in installer rehearse job, suggest to revert the change in this yaml file.